tests/benchdnn/graph/custom_driver.cpp

```diff
@@ -269,6 +271,102 @@ int execute(const prb_t *prb, const args_t &args, res_t *res) {
 }
 } // namespace reshape
 
+namespace softmax {
```

Comment:
It's a placeholder for the idea to reuse benchdnn's softmax prb discussed offline.